app/src/main/java/org/jboss/hal/client/configuration/subsystem/elytron/OtherSettingsView.java

```diff
@@ -256,6 +258,48 @@
                 .build()
                 .create();
 
+        aggregateSecurityEventListenerView = new ResourceView.Builder(tableButtonFactory, primaryIdLogs,
+                Ids.ELYTRON_AGGREGATE_SECURITY_EVENT_LISTENER, "Aggregate Security Event Listener",
+                AGGREGATE_SECURITY_EVENT_LISTENER_ADDRESS, this,
+                () -> presenter.reload())
+                .setNavigation(navigation)
+                .setMetadataRegistry(metadataRegistry)
+                .setTableAddCallback((name, address) -> presenter.reload())
+                .build()
+                .create();
```

Comment:
The builder constructs a ResourceView.Builder() class, that is a builder preparation, them the ResourceView.create() creates all necessary table, forms, tabs, pages then add them to layout.